planning/behavior_path_planner/src/planner_manager.cpp

```diff
@@ -636,15 +636,21 @@ void PlannerManager::resetRootLanelet(const std::shared_ptr<PlannerData> & data)
 
   const auto root_lanelet = updateRootLanelet(data);
 
+  // if root_lanelet is not route lanelets, reset root lanelet.
+  // this can be caused by rerouting.
+  const auto route_handler = data->route_handler;
```

Comment:
```suggestion
  const auto & route_handler = data->route_handler;
```
I think `&` is okay for this case.

Reply:
thanks!